sovrin_client/test/anon_creds/test_public_repo.py

```diff
@@ -112,14 +114,16 @@ def testGetSchemaBySeqNo(submittedSchemaDefGvt, publicRepo, looper):
 
 
 def testGetSchemaByInvalidSeqNo(submittedSchemaDefGvt, publicRepo, looper):
```

Comment:
Yes, with this fix the CLI shows errors with a wrong `ref` argument properly. Wrote the tests verifying the CLI output in such cases.